airflow/providers/google/provider.yaml

```diff
@@ -89,6 +89,12 @@ dependencies:
   - apache-airflow>=2.6.0
   - apache-airflow-providers-common-sql>=1.7.2
   - asgiref>=3.5.2
+  # When upgrading the major version of gcloud-aio-auth we want to make sure to
```

Comment:
Is something block us from upgrading version now?

Reply:
Generally speaking, no. I suppose a bit more care is needed when doing so, as it affects a range of async code of the provider.